### DISCUSSION TITLE
File

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#727] [Richieste di aiuto] [answered]
Salve! Come capite il problema dal file che vi viene allegato per problemi nel registro elettronico?

**Comment by trinko** [comment, marked as answer]
Bisogna cercare la riga con indicato un errore critico (**CRITICAL**): qui viene indicato il file e la riga dove si è generato l'errore. Ad esempio:
```
at /src/Util/BachecaUtil.php:162
```
Questo, assieme alla descrizione dell'errore riportata in inglese da' delle indicazioni su ciò che è successo. Naturalmente bisogna anche conoscere come funziona l'applicazione.
